lib/tasks/temporary/hacktoberfest_badges.rake

```diff
@@ -0,0 +1,23 @@
+# TODO: Will be removed my @mstruve once badges are rewarded for 2020
+task award_hacktoberfest_badges: :environment do
+  badge_id = Badge.find_by(slug: "hacktoberfest-2020")&.id
+
+  User.where.not(github_username: nil).includes(:identities).find_each do |user|
+    identity = user.identities.github.first
+    next unless identity
+
+    github_user_id = identity.uid
+    response = HTTParty.get(
+      "https://hacktoberfest.digitalocean.com/api/state/#{github_user_id}",
+      headers: { "Authorization" => ENV["HACKTOBERFEST_AUTH_KEY"] },
+    )
+    if response # contains completed Hacktoberfest indicator then award the badge
```

Comment:
This if will always be trueish. 

```ruby
[1] pry(main)> HTTParty.get("https://hacktoberfest.digitalocean.com/api/state/foobar")
=> "not-authorized"
```

it's better to check the return code:

```ruby
[2] pry(main)> response = HTTParty.get("https://hacktoberfest.digitalocean.com/api/state/foobar")
=> "not-authorized"
[3] pry(main)> response.code
=> 401
[4] pry(main)> response.ok?
=> false
```

Reply:
This is not done, we will have to check the response for an indicator that Hacktoberfest was completed, not sure yet if that will just be a status code or a value in the response hash I need to check